route_network/src/route_network/planner.py

```diff
@@ -45,7 +45,11 @@
 PKG = 'route_network'
 import roslib; roslib.load_manifest(PKG)
```

Comment:
Please remove `import roslib; roslib.load_manifest...` and the PKG var as well, it is not needed since we moved to catkin.